### ISSUE TITLE
Is it possible to use tiling to reduce gpu memory for training and testing ?

### ISSUE THREAD

**Issue by Michaelwhite34** [abdallahdib/DeepNextFace#1]

**Comment by abdallahdib** [comment]
yes i think this is possible, (https://github.com/BachiLi/redner/blob/master/pyredner/camera.py) by using the viewport parameters to render the scene in tiles. this could solve the issue of gpu memory. i will try to add this
